packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -299,8 +314,7 @@ export function createPlugin(optimizerOptions: OptimizerOptions = {}) {
         if (typeof opts.srcDir === 'string' && !fs.existsSync(opts.srcDir)) {
           throw new Error(`Qwik srcDir "${opts.srcDir}" not found.`);
         }
-        for (const alias in opts.input) {
-          const input = opts.input[alias];
+        for (const [_, input] of Object.entries(opts.input)) {
```

Comment:
Use `Object.values`, put it in a const and then you can use it instead of firstInput too.

Reply:
I can do that in a global variable (let instead of const), but I need to make the Array.isArray verification and the assignment in every case, because the inputs are used in three different functions.

Reply:
yeah, I think this is fine. any more changes it seems like a larger refactor